guidelines/dxp/code_review.md

```diff
@@ -22,10 +22,9 @@ These are guidelines that the Frontend Infrastructure Team follows when developi
     Note that if you create PRs with [the `gh` command-line tool](https://www.npmjs.com/package/gh) you obviously can't see or check a checkbox. In that case, you may wish to add the people who most frequently review your code as [collaborators to your repo](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/inviting-collaborators-to-a-personal-repository), as well as the team members who tend to be generally involved in code review:
 
     -   [bryceosterhaus](https://github.com/bryceosterhaus)
-    -   [jbalsas](https://github.com/jbalsas)
-    -   [jonmak08](https://github.com/jonmak08)
+    -   [javierdearcos](https://github.com/javierdearcos)
     -   [markocikos](https://github.com/markocikos)
-    -   [wincent](https://github.com/wincent)
+    -   [matuzalemsteles](https://github.com/matuzalemsteles)
```

Comment:
shouldn't nhpatt be here too?

Reply:
I thought that if he was stepping into your role he might want to delegate. But if he wants to be in the list, he definitely can be!

`@nhpratt` 😂 — speak your mind!

Reply:
Yeah, let me in! :D

Reply:
I knew @nhpatt wouldn't be able to resist the urge...

![more](https://user-images.githubusercontent.com/905006/116674686-dea0bb00-a99c-11eb-97f1-d86d4616adc9.gif)

Reply:
> @nhpratt 😂 — speak your mind!

![snap](https://user-images.githubusercontent.com/905006/116674792-fd9f4d00-a99c-11eb-8955-ae59db41435a.gif)

Reply:
I'm going to change my nickname in twitter to @nhpratt just to have a mention by the great @wincent 

![](https://media.giphy.com/media/xT5LMqkUezumMOKS88/giphy.gif)